src/app/pages/reports-dashboard/components/report/report.component.ts

```diff
@@ -222,6 +223,13 @@ export class ReportComponent extends WidgetComponent implements AfterViewInit, O
     });
   }
 
+  async ngOnInit(): Promise<void> {
+    const zoom = this.zoomLevels[this.timeZoomIndex];
+    const rrdOptions = await this.convertTimespan(zoom.timespan);
```

Comment:
I'm surprised await works here. I would imagine that Angular doesn't wait for promises in ngOnInit to resolve.